src/tokens/functional/color/dark/primitives-dark.json5

```diff
@@ -34,8 +34,8 @@
         },
       },
       mix: {
-        color: '{base.color.neutral.4}',
-        weight: 0.25,
+        color: '{base.color.neutral.3}',
+        weight: 0.95,
```

Comment:
@smockle since this so close to 100%, did you try to just remove the mix all together?

Reply:
I did, but it’s not quite light enough: `#8b949e` (`base-color-neutral-3`) on `#292e36` is 4.45:1.

Reply:
If we _also_ adjust [`control.transparent.bgColor.hover`’s `alpha`](https://github.com/primer/primitives/blob/44c86c0d327cef5a7180baec9be614222ac8fdc7/src/tokens/functional/color/dark/patterns-dark.json5#L298) from 12% down to 11%, then `base.color.neutral.3` (without mixing) is sufficient.